packit/copr_helper.py

```diff
@@ -219,29 +248,31 @@ def create_copr_project_if_not_exists(
                 failure_message = (
                     f"Copr project update failed for '{owner}/{project}' project."
                 )
-                logger.info(f"Updating copr project '{owner}/{project}'")
-                for field, (old, new) in fields_to_change.items():
-                    logger.debug(f"{field}: {old} -> {new}")
-                    kwargs: dict[str, Any] = {
-                        arg_name: new
-                        for arg_name, (old, new) in fields_to_change.items()
-                    }
-                    logger.debug(f"Copr edit arguments: {kwargs}")
-                    self.copr_client.project_proxy.edit(
-                        ownername=owner,
-                        projectname=project,
-                        **kwargs,
-                    )
+                self.update_copr_project(owner, project, fields_to_change)
 
-            fields_to_change = None
             failure_message = (
```

Comment:
This `failure_message` overriding the previous one is a bit hard to read even though I think it is working and you could spare a try/except clause in this way.